Wikipedia/Code/WMFExploreCollectionViewCell.m

```diff
@@ -4,26 +4,17 @@
 @implementation WMFExploreCollectionViewCell
 
 - (UICollectionViewLayoutAttributes *)preferredLayoutAttributesFittingAttributes:(UICollectionViewLayoutAttributes *)attributesToFit {
-    if ([attributesToFit isKindOfClass:[WMFCVLAttributes class]] && [(WMFCVLAttributes *)attributesToFit precalculated]) {
-        return attributesToFit;
```

Comment:
Bailing here seemed to prevent the size from being calculated - maybe when cells were recycled?